content/blog/2021-10-05-extending-models-with-more-data.md

```diff
@@ -0,0 +1,256 @@
+shifts. Now you have this model that has been going through the training process
+with a specific dataset and you need to make the model more generic.
+
+There's likely something that your model learned that can be useful on this new
+dataset, so you might not have to restart the entire training process. We'll do
+an example of updating a pre-trained model to use a broader dataset with DVC. By
+the end of this, you should see how you can handle this quickly and start
+running new experiments to get a more generic model.
+
+## The original pre-trained model
+
+For this post, we'll be making a more generic image classifier by taking the
+original dataset with bees and ants and adding cats and dogs to it. You can
+clone [this GitHub repo](https://github.com/iterative/pretrained-model-demo) to
+get the current bees and ants model and check out
+[this post](https://dvc.org/blog/transfer-learning-experiments) on how we
+experimented with both AlexNet and SqueezeNet to build this model.
+
+So we're starting from our current bees and ants model and extending it to
+classify dogs and cats as well. We'll start by adding some cats and dogs data to
+our validation data and do some experiments with the current model to see how it
+performs on generic data.
+
+Then we'll add the cats and dogs data to the training data and watch how the
+model improves as we run experiments.
+
+## Updating the dataset with DVC
+
+To add the new cats and dogs dataset to the project, we'll use this DVC command.
+
+```dvc
+dvc get https://github.com/iterative/dataset-registry blog/cats-dogs
+```
+
+This downloads a sample dataset with images of cats and dogs. You can use this
+command to download files or directories that are tracked by DVC or Git. This
+command can be used from anywhere in the file system, as long as DVC is
+installed.
+
+This will make a new directory called `./cats-dogs/data/` that was downloaded
+from the DVC remote and it has images for cats and dogs. Now we can slowly add
+in the new data to the existing data.
+
+We'll start by moving the `val` data for `cats` and `dogs` from the
+`/cats-dogs/data/` directory to the corresponding directory in
+`data/hymenoptera_data`.
+
+_Just a quick note, cats and dogs don't really belong in the `hymenoptera`
+directory since that's specific to ants and bees, but it's the easiest and
+fastest way to add the data for this tutorial._
+
+With this new data in place, we can start training our model.
+
+## Running new experiments with generic data
+
+With the updated data, let's run an experiment on the model and see how good the
+results are. To run a new experiment, open your terminal and make sure you have
+a virtual environment enabled. Then run this command:
+
+```dvc
+dvc exp run
+```
+
+Once the training epochs are finished, run the following command.
+
+```dvc
+dvc exp show --no-timestamp --include-metrics step,acc,val_acc,loss,val_loss --include-params lr,momentum
```

Comment:
use `\` to split this long command

Reply:
we should create a way to save table filters and sorting I think in DVC, wdyt @dberenbaum ?

Reply:
I extracted this point to https://github.com/iterative/dvc/issues/6708. We plan to tackle these kind of improvements soon, but there are a lot of ideas that we need to sort through and prioritize.